File: python/Isograms - 7 kyu.py
def is_isogram(string):
    
    #same case for case insensitivity
    lower_string = string.lower()
    
    #chars we have seen
    chars = []
    
    for i in lower_string:
        
        #checking if we have iterated
        if i in chars:
            return False
        
        chars.append(i)
    
    return True

File: c++/Isograms - 7 kyu.cpp
#include <iostream>
#include <string>

using namespace std;

bool is_isogram(string word) {
  bool seen[26] = {false};
  for (char letter : word) {
    letter = tolower(letter);
    if (seen[letter - 'a']) {
      return false;
    }
    seen[letter - 'a'] = true;
  }
  return true;
}
